adminservice/src/main/resources/templates/whitelisted-numbers.vm

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head><title>Whitelisted numbers</title></head>
+<body>
+	<h2>Add whitelisted number</h2>
+	<form action="" method="post">
+        <p><label>Number: <input type="tel" name="mobileNumber" title="e.g. +31612345678" placeholder="+31612345678" pattern="\+[1-9][0-9]+" /></label></p>
+        <p><input type="submit" value="Submit" /></p>
+    </form>
+    
+    <h2>Current whitelisted numbers</h2>
```

Comment:
Change in one of:
* Currently white-listed numbers
* Current set of white-listed numbers